Tests/MapboxMobileEventsTests/MMEAPIConfigTests.m

```diff
@@ -95,6 +95,7 @@ - (void) test004_ShortUpdateInterval {
     XCTAssertNil(configError);
 }
 
+#if !SWIFT_PACKAGE
```

Comment:
What is the problem with that test? Worth adding a comment?

Reply:
Support for resources was added in Swift 5.3 but we currently only require 5.2. 5.2 is included in Xcode 11 and 5.3 in 12.